spec/latest/core/index.html

```diff
@@ -1266,6 +1266,35 @@ <h3>Description of HTTP Status Codes and Errors</h3>
       }
       -->
     </pre>
+
+    <p>It is also possible to provide an [[!RFC7807]] compatible</a> error description.
+      To do so, server should provide a <i>application/problem+json</i> response with additional header
+      pointing to <i>http://www.w3.org/ns/hydra/error.context.jsonld</i> context.</p>
```

Comment:
```suggestion
      pointing to <i>http://www.w3.org/ns/hydra/error/context.jsonld</i> context.</p>
```

Reply:
is this supposed to resolve?

Reply:
Once it's merged I guess

Reply:
Yes - same way as the core vocabulary context.

Reply:
Latest changes are using `http://www.w3.org/ns/hydra/error` which should resolve with content negotiation  - is that acceptable for you @tpluscode ?